applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -137,10 +163,17 @@ static void stream_stopped_cb(struct bt_audio_stream *stream)
 
 	LOG_INF("Audio Stream %p stopped", (void *)stream);
 
-	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_NOT_STREAMING);
-	ERR_CHK(ret);
+	if (stream->conn == headset_conn[AUDIO_CHANNEL_LEFT]) {
+		atomic_clear(&iso_tx_pool_alloc[AUDIO_CHANNEL_LEFT]);
```

Comment:
Hmmm, for BIS we figured out that we should not do atomic_clear in the stopped_cb, since decrementing shall be handled in the sent_cb.
This should be the same for CIS?

Reply:
If there are two packets for left channel in the controller are waiting to be sent, then iso_tx_pool_alloc[LEFT] will be 2.
But if disconnection happen at the moment, the buffer in the controller will flush, so we also need to clean the counter in here.